app/views/front.html

```diff
@@ -12,18 +12,19 @@
 
     <base href="/">
 
+    <link rel="stylesheet" href="/styles/enfants.css">
+    <link rel="stylesheet" href="/styles/footer.css">
+    <link rel="stylesheet" href="/styles/foyer/capture-montant-ressource.css">
```

Comment:
No! CSS files are sorted on purpose: two identical selectors will be discriminated based on their declaration order. You must not sort them alphabetically, but by specificity.